docs/book/src/plugins/available/kustomize-v2.md

```diff
@@ -0,0 +1,105 @@
+# Kustomize v2
+
+**(Default Scaffold)**
+
+The Kustomize plugin allows you to scaffold all kustomize manifests used
+with the language base plugin `base.go.kubebuilder.io/v4`.
+This plugin is used to generate the manifest under the `config/` directory
+for projects built within the `go/v4` plugin (default scaffold).
+
+Projects like [Operator-sdk][sdk] use the Kubebuilder project as a library
+and provide options for working with other languages such as Ansible and Helm.
+The Kustomize plugin helps them maintain consistent configuration across
+languages. It also simplifies the creation of plugins that perform
+changes on top of the default scaffold, removing the need for manual
+updates across multiple language plugins. This approach allows the
+creation of "helper" plugins that work with different projects
+and languages.
+
+<aside class="note">
+<h1>Examples</h1>
+
+You can check the kustomize content by looking at the `config/` directory provided in the sample `project-v4-*`
+under the [testdata][testdata] directory of the Kubebuilder project.
+
+</aside>
+
+## How to use it
+
+If you want your language plugin to use kustomize, use the [Bundle Plugin][bundle] to specify that your language plugin is composed of your language-specific plugin and kustomize for its configuration, as shown:
+
+```go
+import (
+   ...
+   kustomizecommonv2 "sigs.k8s.io/kubebuilder/v3/pkg/plugins/common/kustomize/v2"
```

Comment:
for kustomize, its `v3`?

Reply:
Good point it should be v4 now. (sigs.k8s.io/kubebuilder/v4/pkg/plugins/common/kustomize/v2)